LiteDB/Engine/Pages/BasePage.cs

```diff
@@ -233,7 +233,6 @@ public void MarkAsEmtpy()
             this.IsConfirmed = false;
 
             // block information
-            this.ItemsCount = 0;
```

Comment:
dublicate

Reply:
This has been fixed in the meantime by removing the other ItemCount assignment. If you remove this one as well, we'll be introducing a bug. Could undo this change?

Reply:
done